MuvrKit/MKWatchConnectivity.swift

```diff
@@ -51,6 +60,7 @@ struct MKConnectivitySessions {
         if let oldProps = sessions[session] {
             if oldProps.ended && !newProps.ended { fatalError("Session resurrection") }
             sessions[session] = newProps
+            saveSessions(sessions)
```

Comment:
why not having only one of the update function doing the work and the second calling that one ?